.changeset/tired-papers-speak.md

```diff
@@ -0,0 +1,5 @@
+---
+"docs": patch
+---
+
+Explain where to generate static tokens in authentication reference.
```

Comment:
Changesets should be expressed in past tense when possible and end without a period
```suggestion
Added explaination about where to generate static tokens in authentication reference
```